cmd/eval.go

```diff
@@ -374,7 +380,64 @@ func testTypoSuggestFn(cmd *cobra.Command, args []string) {
 		} else {
 			log.Infof("No suggest for '%s'.", e.Query)
 		}
+
+		if !checkConstantTerms(e.Query, res.String) {
+			constantTermsError++
+		}
+	}
+
+	log.Infof(
+		"\n\nNot found suggests for known typos: %d from %d."+
+			"\nProbable False Positive: %d from %d."+
+			"\nConstant terms errors: %d.",
+		noSuggestForTypoCnt,
+		len(typos),
+		falsePositiveCnt,
+		len(evalSet),
+		constantTermsError,
+	)
+}
+
+func checkConstantTerms(request string, response string) bool {
```

Comment:
What are constant terms? What this function checks?

Reply:
constant terms = terms that we ignore in typo check (currently it is numeric values but later it could be something else)

Reply:
The function is used to test that the constant terms logic is working correctly for all kind of queries